Array_Rotation/arrayrotation.c

```diff
@@ -0,0 +1,37 @@
+#include<stdio.h>
```

Comment:
Please change the file name to  Array_Rotation/Array_Rotation.c

Reply:
@sandhyabhan Please change the file name to Array_Rotation.c not ArrayRotation.c

Reply:
I tried doing so...but it is showing error that file name already existed.
what should i do?